codeguru_profiler_agent/profiler_disabler.py

```diff
@@ -6,7 +6,7 @@
 logger = logging.getLogger(__name__)
 CHECK_KILLSWITCH_FILE_INTERVAL_SECONDS = 60
 MINIMUM_MEASURES_IN_DURATION_METRICS = 20
-MINIMUM_SAMPLES_IN_PROFILE = 5
+MINIMUM_SAMPLES_IN_PROFILE = 1
```

Comment:
I know you got this as a comment from @pandpara , but I don't think this should be updated, and especially in this commit. This affects if the profiler is disabled or not and it's different than what the general message of this commit is about. If there's a concern about this value, we can do it in a different commit (even release).